packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_messages.dart

```diff
@@ -0,0 +1,168 @@
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final colorScheme = theme.colorScheme;
+    TextSpan colorA(String text) => _coloredSpan(
+      text,
+      context: context,
+      color: colorScheme.declarationsSyntaxColor,
+    );
+    TextSpan colorB(String text) => _coloredSpan(
+      text,
+      context: context,
+      color: colorScheme.modifierSyntaxColor,
+    );
+    TextSpan colorC(String text) => _coloredSpan(
+      text,
+      context: context,
+      color: colorScheme.variableSyntaxColor,
+    );
+    TextSpan colorD(String text) => _coloredSpan(
+      text,
+      context: context,
+      color: colorScheme.controlFlowSyntaxColor,
+    );
+    TextSpan colorE(String text) => _coloredSpan(
+      text,
+      context: context,
+      color: colorScheme.stringSyntaxColor,
+    );
+    TextSpan colorF(String text) => _coloredSpan(
+      text,
+      context: context,
+      color: colorScheme.functionSyntaxColor,
+    );
+    TextSpan colorG(String text) => _coloredSpan(
+      text,
+      context: context,
+      color: colorScheme.numericConstantSyntaxColor,
+    );
+    TextSpan highlight(TextSpan original) => _highlight(
+      original,
+      highlighterColor:
+          theme.isDarkTheme ? _darkHighlighterColor : _lightHighlighterColor,
+    );
+
+    return Text.rich(
+      TextSpan(
+        children: [
+          const TextSpan(text: '\nPlease move your cursor anywhere inside a '),
+          TextSpan(
+            text: 'Flutter widget constructor invocation',
+            style: theme.boldTextStyle,
+          ),
+          const TextSpan(text: ' to view and edit its properties.\n\n'),
+          const TextSpan(
+            text:
+                'For example, the highlighted code below is a constructor invocation of a ',
+          ),
+          TextSpan(
+            text: 'Text',
+            style: Theme.of(
+              context,
+            ).fixedFontStyle.copyWith(color: colorScheme.primary),
+          ),
+          const TextSpan(text: ' widget:\n\n'),
+          colorA('@override\n'),
+          colorB('Widget '),
+          colorG('build'),
+          colorC('('),
+          colorB('BuildContext '),
+          colorA('context'),
+          colorC(') '),
+          colorC('{\n'),
+          colorD(' return '),
+          highlight(colorB('Text')),
+          highlight(colorC('(\n')),
+          highlight(colorE('  "Hello World!"')),
+          highlight(colorF(',\n')),
+          highlight(colorA('  overflow')),
+          highlight(colorF(': ')),
+          highlight(colorB('TextOveflow')),
+          highlight(colorF('.')),
+          highlight(colorG('clip')),
+          highlight(colorF(',\n')),
+          highlight(colorC('  )')),
+          highlight(colorF(';\n')),
+          colorC('}'),
+        ],
+      ),
+    );
+  }
+
+  TextSpan _coloredSpan(
+    String text, {
+    required BuildContext context,
```

Comment:
instead of passing the context, consider passing in the fontstyle so that the lookup `Theme.of(context).fixedFontStyle` only has to happen once instead of once for each call to this method

Reply:
Done!